openassessment/workflow/models.py

```diff
@@ -608,27 +612,35 @@ def is_cancelled(self):
         """
         return self.cancellations.exists()
 
+    @property
+    def identifying_uuid(self):
+        """
+        Returns the primary identifying uuid for this workflow.
+        AssessmentWorkflow returns submission_uuid, and TeamAssessmentWorkflow returns team_submission_uuid
```

Comment:
I think line 619 (second comment line) can be removed as it is documented in code - thoughts? I would make a mention, though, that this method can be overridden by derived classes.